osd/aws/sts_deleted_provider.json

```
@@ -0,0 +1,7 @@
+{
+  "severity": "Error",
+  "service_name": "SREManualAction",
+  "summary": "Action required: Fix AWS Role to maintain SRE support",
```

This subject line looks like it needs updating to reflect the description body. Maybe something like 
```suggestion
  "summary": "Action required: Review OpenIDConnect provider configuration",
```